website/src/pages/index.tsx

```diff
@@ -33,6 +33,21 @@ const testimonials: Testimonial[] = [
             target: '_blank',
         },
     },
+    {
+        customer: 'SoFi',
+        logo: '/external-logos/sofi-logo-white.png',
+        quote:
+            'Sourcegraph is an ingrained part of my daily process at SoFi. With Sourcegraph, our engineers can understand all of the repercussions of committing a change to a service that is exposed to other services.',
```

Comment:
I'm torn — this is a long quote, and I'm wondering if it would be more impactful with just the 2nd sentence (focused on the use case) or the first (the personal vouch) or sticking with both and just leaving it really long...

Reply:
Lets just take the second half.